### DISCUSSION TITLE
unwanted dagrun trigger after unpausing

### DISCUSSION THREAD

**Discussion by itay-bardugo** [apache/airflow#28539] [Q&A]
Hey.
I have a few dags that 
I paused  them this morning.
After unpausing them a few hours ago, they started running from the time they went paused to now. Even though catchup is set to False.
Start date set to somewhen in the past in 2019, hardcoded.
 i can confirm that the scheduler did not execute dags from previous days, but it invoked the dags from today started from the time the dag switched to pasue.
 i am trying to understand how to disable this behaviour.

What should i do in order to have the next dagrun starting from the time it got unpaused instead of the last time it executed today?

**Comment by potiuk** [comment]
Change Start date. Pausing is not to create "holes" in the timeline, but in order to pause DAG from running. If you want to have "holes" you need to pause the DAG and set the "start_date" for the new starting date.

**Reply by itay-bardugo** [reply]
Hey, thanks for replying.
Everytime that someone from my company pauses a dag i should change the start date programmatically? Does not sound like a scalable solution..

**Reply by potiuk** [reply]
Another use cases is "Every time someone pauses DAG temporarily" and re-enables it they have to manually backfill it - sounds even less scalable.

Also what happens if you backfill the range? Should all of those skipped runs be run or not?

Wnat you are looking at, is completely different use case and feature - not supported in Airlfow for now. You want  to mark certain "date range" of execution  as "excluded". And this feature does not exist.

**Reply by potiuk** [reply]
But this is an interesting feature request  - maybe you would liek to think about it and possibly even implement it (PRs are most welcome).

**Reply by itay-bardugo** [reply]
Sounds like a plan :)

**Reply by Taragolis** [reply]
BTW, we have [LatestOnlyOperator](https://airflow.apache.org/docs/apache-airflow/stable/concepts/dags.html#latest-only) just put it as first task in the DAG. In this case DAG still scheduled but LatestOnlyOperator will skip next task and regardless on trigger_rule it could skip all other task within the DAG

**Reply by itay-bardugo** [reply]
Will check that.
Thanks for assisting!

**Reply by potiuk** [reply]
> BTW, we have [LatestOnlyOperator](https://airflow.apache.org/docs/apache-airflow/stable/concepts/dags.html#latest-only) just put it as first task in the DAG. In this case DAG still scheduled but LatestOnlyOperator will skip next task and regardless on trigger_rule it could skip all other task within the DAG

NICE! TIL @Taragolis :)

**Reply by Taragolis** [reply]
I think also better to have a Short Circuit version of this operator, it should be pretty simple to implement it

**Comment by actinonel** [comment]
UP.
Nothing has changed, this functionality is still not available?
